plugin/plugin.go

```diff
@@ -319,6 +320,8 @@ func (p *OrmPlugin) parseBasicFields(msg *generator.Descriptor) {
 				} else {
 					fieldOpts.Tag = tagWithType(tag, "varchar(48)")
 				}
+			} else if rawType == protoTimeOnly {
+				fieldType = "string"
```

Comment:
Could you also include `type:time` tag to the generated struct field(like we do for other types): it may be useful for those who create their schema with `gorm.AutoMigrate`.
```golang
fieldOpts.Tag = tagWithType(tag, "time")
```